docs/lsp-clients.json

```diff
@@ -496,6 +496,15 @@
     "lsp-install-server": "jsts-ls",
     "debugger": "Yes (Firefox/Chrome)"
   },
+  {
+    "name": "typespec",
+    "full-name": "TypeSpec",
+    "server-name": "tsp server",
+    "server-url": https://github.com/microsoft/typespec/tree/main/packages/compiler",
```

Comment:
it seems that a `"` is missing at url beginning?

Reply:
Good catch! Not sure how I missed that.